scripts/i18n/gen_lang_stubs.py

```
@@ -111,7 +106,7 @@ def main() -> int:
         try:
             with f.open(encoding="utf-8") as fh:
                 data = json.load(fh)
-        except Exception as exc:  # noqa: BLE001
+        except Exception as exc:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)